### PR TITLE
WEBAPP-77: Add label for inputs

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/locales.json [hunk 1]
```diff
@@ -37,29 +37,17 @@
...
     },
     "de": {
-      "pageTitle": "Suche"
+      "pageTitle": "Suche",
+      "searchCategory": "Suche nach einer Kategorie"
     },
     "en": {
-      "pageTitle": "Search"
+      "pageTitle": "Search",
+      "searchCategory": "Search a category"
```
Comment (NKlug) [inline src/locales.json]: Search _for_ a category?

File: src/locales.json [hunk 2]
```diff
@@ -130,11 +118,13 @@
...
       "where": "Wo bist du?",
-      "pageTitle": "Wo bist du?"
+      "pageTitle": "Wo bist du?",
+      "searchCity": "Suche nach deinem Ort"
     },
     "en": {
       "where": "Where are you?",
-      "pageTitle": "Where are you?"
+      "pageTitle": "Where are you?",
+      "searchCity": "Search your city"
```
Comment (NKlug) [inline src/locales.json]: I think it should be 'Search _for_ your city'.

File: src/locales.json (hunk 1 above)
Comment (klinzo) [reply src/locales.json]: yep you're right

File: src/modules/common/components/SearchInput.js [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
   render () {
+    const {onClickInput, filterText, placeholderText} = this.props
     return (
       <Spacer space={this.props.spaceSearch}>
         <Wrapper>
           <SearchIcon />
-          <TextInput placeholder={this.props.t('search')}
-                     defaultValue={this.props.filterText}
+          <TextInput placeholder={placeholderText}
+                     label={placeholderText}
```
Comment (maxammann) [inline src/modules/common/components/SearchInput.js]: Is label really used? https://github.com/Integreat/integreat-webapp/blob/f67040565993824bbad672d52e49f9cc5b67358a/src/modules/common/components/SearchInput.styles.js#L10

Also a `<input>` does not have a label attribute: https://www.w3schools.com/tags/tag_input.asp

But there is an aria-label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Comment (klinzo) [reply src/modules/common/components/SearchInput.js]: what do you mean by "really used"? Changed to aria-label.

File: src/modules/common/components/__tests__/__snapshots__/ScrollingSearchBox.spec.js.snap [hunk 4]
```diff
@@ -34,7 +35,8 @@ exports[`ScrollingSearchBox should set correct reference 1`] = `
         />
         <input
           class="SearchInputstyles__TextInput-kJgKBC lfoCjL"
-          placeholder="common:search"
+          label="Placeholder"
```
Comment (Schedulaar) [inline src/modules/common/components/__tests__/__snapshots__/ScrollingSearchBox.spec.js.snap]: wrong snapshot, I think you just need to update them.

File: src/modules/common/components/__tests__/__snapshots__/SearchInput.spec.js.snap [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
-  onFilterTextChange={[Function]}
-  t={[Function]}
-/>
+<SearchInputstyles__Spacer>
+  <SearchInputstyles__Wrapper>
+    <SearchInputstyles__SearchIcon />
+    <SearchInputstyles__TextInput
+      autoFocus={true}
+      defaultValue="Test"
+      label="Placeholder"
```
Comment (Schedulaar) [inline src/modules/common/components/__tests__/__snapshots__/SearchInput.spec.js.snap]: Here as well...

File: src/modules/common/components/__tests__/__snapshots__/ScrollingSearchBox.spec.js.snap (hunk 4 above)
Comment (klinzo) [reply src/modules/common/components/__tests__/__snapshots__/ScrollingSearchBox.spec.js.snap]: I was sure I did update them o.O

**Pull request merged**